test/python/circuit/test_controlled_gate.py

```diff
@@ -960,9 +967,9 @@ def test_open_control_ccx_unrolling(self):
         ccx = CCXGate(ctrl_state=0)
         qc.append(ccx, [0, 1, 2])
         dag = circuit_to_dag(qc)
-        with self.assertWarns(DeprecationWarning):
-            unroller = Unroller(["x", "ccx"])
-        unrolled_dag = unroller.run(dag)
+        unroller = UnrollCustomDefinitions(std_eqlib, ["x", "ccx"])
```

Comment:
How about introducing a seperate function for the equality check, like in the other test? I.e.
```
def assertEqualTranslated(self, circuit, unrolled_reference, basis):
        unroller = UnrollCustomDefinitions(std_eqlib, basis)
        basis_translator = BasisTranslator(std_eqlib, basis)
        unrolled = basis_translator(unroller(circuit))  
        self.assertEqual(unrolled, unrolled_reference)
```
Note that we can also skip the explicit conversion to `DAGCircuit`

Reply:
Sounds good, will do.